man/tpm2_quote.1.md

```diff
@@ -41,7 +41,12 @@
 
   * **-o**, **--outFile**:
 
-    Output file path, recording the two structures output by tpm2_quote function.
+    Output file path, recording the two structures (quote and signature) output by tpm2_quote function.
+    Contains only one structure (quote) when -s is also specified
+
+  * **-s**, **--outSigFile**:
```

Comment:
Please don't use camel case for neither options nor variables. I know that you are just being consistent with the current implementation but that's something we want to change project wide.

Reply:
No problem. I changed the log opt name.
Also note that someone already added a -f option equivalent to my proposed -s, so I dropped the first patch in this series. I'll resubmit the -G part.